docs/topics/tour/kotlin-tour-control-flow.md

```diff
@@ -66,6 +69,8 @@ fun main() {
 Use `when` when you have a conditional expression with multiple branches.
 `when` can be used either as a statement or as an expression.
 
+A **statement** doesn't return anything but performs actions instead.
+
 Here is an example of using `when` as a statement:
 * Place the conditional expression within parentheses `()` and the actions to take
```

Comment:
This sentence may give the wrong impression that it only applies to the case in which `when` is used as a statement (even though you later mention the same syntax for using as an expression).

Reply:
Section reworked!